docs/proton-kafka.md

```diff
@@ -41,57 +39,67 @@ SETTINGS
+#### type
 
-:::
-### DDL Settings
+Must be set to `kafka`. Compatible with:
 
-#### type
-Need to be `kafka`. This works with Apache Kafka, Redpanda, Confluent Platform or Cloud,and many other Kafka compatible message platforms.
+* Apache Kafka
+* Confluent Platform or Cloud
+* Redpanda
+* Other Kafka-compatible systems
 
 #### brokers
-One or more brokers with ports.
+
+Comma-separated list of broker addresses (host\:port), e.g.:
+
+```
+kafka1:9092,kafka2:9092,kafka3:9092
+```
 
 #### topic
-One external stream will connect to one topic.
+
+Kafka topic name to connect to.
 
 #### security_protocol
+
 The supported values for `security_protocol` are:
 
 - PLAINTEXT: when this option is omitted, this is the default value.
 - SASL_SSL: when this value is set, username and password should be specified.
-  - If you need to specify own SSL certification file, add another setting `ssl_ca_cert_file='/ssl/ca.pem'` New in Proton 1.5.5, you can also put the full content of the pem file as a string in the `ssl_ca_pem` setting if you don't want to, or cannot use a file path, such as on Timeplus Enterprise or in Docker/Kubernetes environments.
-  - Skipping the SSL certification verification can be done via `SETTINGS skip_ssl_cert_check=true`.
+  - If users need to specify own SSL certification file, add another setting `ssl_ca_cert_file='/ssl/ca.pem'`. Users can also put the full content of the pem file as a string in the `ssl_ca_pem` setting.
+  - To skip the SSL certification verification: `skip_ssl_cert_check=true`.
 
 #### sasl_mechanism
+
 The supported values for `sasl_mechanism` are:
 
-- PLAIN: when you set security_protocol to SASL_SSL, this is the default value for sasl_mechanism.
+- PLAIN: when setting security_protocol to SASL_SSL, this is the default value for sasl_mechanism.
 - SCRAM-SHA-256
 - SCRAM-SHA-512
-- AWS_MSK_IAM: this is available since Timeplus Enterprise 2.7.0 and Proton 1.6.12. Set to this value if you are using AWS MSK with IAM authentication and the EC2 instance or Kubernetes pod has the proper IAM role to access the Kafka topic.
+- AWS_MSK_IAM (for AWS MSK IAM role-based access)
 
-#### username
-Required when `sasl_mechanism` is set to value other than `PLAIN` or `AWS_MSK_IAM`.
+#### username / password
 
-#### password
-Required when `sasl_mechanism` is set to value other than `PLAIN` or `AWS_MSK_IAM`.
+Required when `sasl_mechanism` ise set to SCRAM-SHA-256 or SCRAM-SHA-512. 
 
-Since [Timeplus Enterprise v2.7](/enterprise-v2.7), you can also use the [config_file](#config_file) setting to specify the username and password in a separate file.
+Alternatively, use [`config_file`](#config_file) to securely pass credentials.
 
 #### config_file
-The `config_file` setting is available since Timeplus Enterprise 2.7. You can specify the path to a file that contains the Kafka configuration settings. The file should be in the format of `key=value` pairs, one pair per line. For example:
```

Comment:
This feature is not available in Timeplus Proton and not in 2.6 or earlier TPE. I recommend we still keep it

Reply:
There are lots features which are not available in Proton which introduces so much burn to us and lots of confusions to users as well. We simply didn't / can't track them well already